src/GafferSceneUI/SceneView.cpp

```diff
@@ -2120,9 +2120,9 @@ void SceneView::contextChanged( const IECore::InternedString &name )
 		m_sceneGadget->setSelection( ContextAlgo::getSelectedPaths( getContext() ) );
 		return;
 	}
-	else if( ContextAlgo::affectsExpandedPaths( name ) )
+	else if( ContextAlgo::affectsVisibleSet( name ) )
 	{
-		m_sceneGadget->setExpandedPaths( ContextAlgo::getExpandedPaths( getContext() ) );
+		m_sceneGadget->setVisibleSet( ContextAlgo::getVisibleSet( getContext() ) );
```

Comment:
This line won't compile without the next commit, so I think it's worth just smooshing it in with that commit. It's necessary for every commit to be compilable so that we can use `git bisect` to track down the source of bugs.

Reply:
Noted, and thanks for explaining. Squashed in with a67e1ed.